Foundation/NSGeometry.swift

```diff
@@ -9,10 +9,27 @@
 
 #if os(OSX) || os(iOS)
     import Darwin
-#elseif os(Linux)
+#elseif os(Linux) || os(Android)
     import Glibc
 #endif
 
+#if os(Android)
+private func floor( value: CGFloat.NativeType ) -> Double {
+    return floor( Double(value) )
+}
+private func ceil( value: CGFloat.NativeType ) -> Double {
+    return ceil( Double(value) )
+}
+private func round( value: CGFloat.NativeType ) -> Double {
+    return round( Double(value) )
+}
+#if false
```

Comment:
this is likely generating incorrect code since the host OS is leaking into the build product (per my commentary on the triples) where it is building with the flags of 64 bit but emitting 32 bit. Which is probably incorrect.